DQM/SiPixelCommon/python/SiPixelOfflineDQM_source_cff.py

```diff
@@ -31,6 +31,10 @@
 from DQM.SiPixelMonitorTrack.SiPixelMonitorEfficiency_cfi import *
 SiPixelHitEfficiencySource.saveFile = False
 SiPixelHitEfficiencySource.trajectoryInput = cms.InputTag('generalTracks') 
+from DQM.TrackerMonitorTrack.SiPixelMonitorTrackResiduals_cfi import *
+SiPixelMonitorTrackResiduals.trajectoryInput = 'generalTracks'
+SiPixelMonitorTrackResiduals.Tracks          = 'generalTracks'
+SiPixelMonitorTrackResiduals.Mod_On          = False
```

Comment:
either modify the CFI with these changes or clone the module and give it a new name before changing its configuration
@schneiml @deguio